tests/congruence_tests/test_info.py

```diff
@@ -0,0 +1,16 @@
+from tests.congruence_tests.test_common import init_local, init_remote
+
+
+def test_info():
+    local_client = init_local()
+    remote_client = init_remote()
+    remote_grpc_client = init_remote(prefer_grpc=True)
+
+    local_info = local_client.info()
+    rest_info = remote_client.info()
+    grpc_info = remote_grpc_client.info()
+
+    assert local_info.title == rest_info.title
+    assert local_info.version is not None
+    assert rest_info.version == grpc_info.version
+    assert rest_info == grpc_info
```

Comment:
Adding the following local mode check, failed the test for me.
I realized the `dev` branch version is still at `1.4.0`. Works well otherwise.


```suggestion
    assert rest_info == grpc_info
    assert rest_info.version == local_info.version
```